cognite/client/_api/functions.py

```diff
@@ -1309,17 +1342,26 @@ def create(
             # We serialize the object as we mutate `item` using the result from _get_function_internal_id.
             item = FunctionScheduleWrite._load(name.dump())
         identifier = _get_function_identifier(item.function_id, item.function_external_id)
+        if item.function_external_id is not None:
+            warnings.warn(
+                "function_external_id is not supported. Replacing it with function_id instead.",
```

Comment:
The code looks to still be supporting it (via ID lookup) so the wording of this needs to be changed accordingly.